BUILD.md

```diff
@@ -151,36 +153,36 @@ The steps below will set up a Python virtual environment to run Avalon.
    If the virtual environment for the current shell session is activated,
    you will the see this prompt: `(_dev)`
 
-10. Install PIP3 packages into your Python virtual environment:
+9. Install PIP3 packages into your Python virtual environment:
 
-    ```bash
-    pip3 install --upgrade setuptools json-rpc py-solc-x web3 colorlog twisted wheel toml
-    ```
+   ```bash
+   pip3 install --upgrade setuptools json-rpc py-solc-x web3 colorlog twisted wheel toml
+   ```
 
-11. Build and install Avalon components:
+10. Build and install Avalon components:
 
     ```bash
     make clean
     make
     ```
 
-12. Build the Avalon SDK Python module:
+11. Build the Avalon SDK Python module:
```

Comment:
Steps 11 (building Avalon SDK python module), 12(building shared_kv), 13(Building Avalon listener) should be removed from here. Step 10 above takes care of that.

Another point (not a problem with your changes) is that in tools/rebuild.sh, we should use "make && make install"

Reply:
That was added recently (adding sdk, shared_kv_storage, and listener) to `tools/rebuild.sh`
I will try that tomorrow.

Reply:
Done